app/models/user.rb

```diff
@@ -14,6 +14,10 @@ def confirmed?
     !!oauth_api_gouv_id
   end
 
+  def full_name
+    "#{last_name} #{first_name}"
```

Comment:
je me demande si ca fait pas bizarre ici d'avoir cet ordre la meme dans les emails. J aurais inverse a vue de nez, peut etre la suite du code me donnera tort

Reply:
autre point, le last_name devrait etre en capslock pour eviter les confusions du type jean daniel et daniel jean dans un meme service

Reply:
> je me demande si ca fait pas bizarre ici d'avoir cet ordre la meme dans les emails. J aurais inverse a vue de nez

No fucking idea, je vais demander à Dorine de trancher. Pour le contexte je pense que ça ne sert qu'à mettre les to/cc dans les emails.

> le last_name devrait etre en capslock pour eviter les confusions du type jean daniel et daniel jean dans un meme service

Là j'aurais aussi tendance à laisser Dorine trancher, même si je m'attends en réalité à ce que DataPass s'occupe de ce formatage

cc @DorineLam

Reply:
On part sur capslock sur last name